package ke.co.xently.remotedatasource.exceptions

class WebsocketConnectionFailedException : WebsocketException("Failed to establish a connection.")